src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -1,32 +1,35 @@
 import { usePagination, useSearch } from '@faststore/sdk'
 import { GatsbySeo } from 'gatsby-plugin-next-seo'
-import React, { useState } from 'react'
+import React, { lazy, Suspense, useState } from 'react'
+import ProductGrid from 'src/components/product/ProductGrid'
 import Filter from 'src/components/search/Filter'
 import Sort from 'src/components/search/Sort'
-import Button, { LinkButton } from 'src/components/ui/Button'
-import SkeletonElement from 'src/components/skeletons/SkeletonElement'
 import FilterSkeleton from 'src/components/skeletons/FilterSkeleton'
-import ProductGrid from 'src/components/product/ProductGrid'
+import SkeletonElement from 'src/components/skeletons/SkeletonElement'
+import Button, { LinkButton } from 'src/components/ui/Button'
 import Icon from 'src/components/ui/Icon'
 
 import Section from '../Section'
-import GalleryPage from './ProductGalleryPage'
 import EmptyGallery from './EmptyGallery'
+import { useDelayedFacets } from './useDelayedFacets'
 import { useGalleryQuery } from './useGalleryQuery'
-import { useOrderedFacets } from './useOrderedFacets'
 
 import './product-gallery.scss'
 
+const GalleryPage = lazy(() => import('./ProductGalleryPage'))
+
 interface Props {
   title: string
 }
 
+const GalleryPageSkeleton = <ProductGrid page={0} pageSize={0} products={[]} />
```

Comment:
Not a big deal but since they're related they could be closer to each other.

```suggestion
const GalleryPage = lazy(() => import('./ProductGalleryPage'))
const GalleryPageSkeleton = <ProductGrid page={0} pageSize={0} products={[]} />

interface Props {
  title: string
}
```